trello_service.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import configparser
+import board as b
```

Comment:
Bit confusing to have a module called "board" which just contains a Card class.

Also you should be able to write e.g. `from board import Card` and then use Card directly instead of b.Card